src/handlers/comment/handlers/index.ts

```diff
@@ -132,7 +132,7 @@ export const userCommands: UserCommands[] = [
   },
   {
     id: IssueCommentCommands.WALLET,
-    description: `<WALLET_ADDRESS | ENS_NAME>: Register the hunter's wallet address. \n  ex1: /wallet 0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266 \n  ex2: /wallet vitalik.eth\n`,
+    description: `<WALLET_ADDRESS | ENS_NAME> <SIGNATURE_HASH>: Register the hunter's wallet address. \n  Your message to sign is: DevPool\n  You can generate SIGNATURE_HASH at https://etherscan.io/verifiedSignatures\n  ex1: /wallet 0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266 0x75329f883590507e581cd6dfca62680b6cd12e1f1665db8097f9e642ed70025146b5cf9f777dde90c4a9cbd41500a6bf76bc394fd0b0cae2aab09f7a6f30e3b31b\n  ex2: /wallet vitalik.eth 0x75329f883590507e581cd6dfca62680b6cd12e1f1665db8097f9e642ed70025146b5cf9f777dde90c4a9cbd41500a6bf76bc394fd0b0cae2aab09f7a6f30e3b31b\n`,
```

Comment:
`DevPool`: Its fine whatever the name is but feel like the name starting with `Ubiquity` sounds better on the marketing side.

Reply:
My friend who is a branding specialist actually advised against branding the DevPool as "Ubiquity DevPool"

Instead he suggested to consider it something brand new. He convinced me of it after several conversations. 

Let's change retroactively if needed.

Reply:
Do you mean we should use `ubiquity/ubiquibot` as a message to sign? @pavlovcik 

Your other suggestion was `UbiquiBot` from https://github.com/ubiquity/ubiquibot/issues/410#issuecomment-1621433888

Reply:
I liked `ubiquity/ubiquibot` more because 
- main repository name
- a little lengthy, will force the user to copy/paste instead of typing. Less chance of mistake.

Reply:
The problem is that this is supposed to be used in the context of any organization's repository so I feel that it's unusual to use specifically this repository's name. 

I vote `DevPool` for now. 

However I like your idea about using a long string so that the user is likely to copy and paste to minimize the chance of typo.

Reply:
Ok. I am marking it resolved for now then.